src/pages-and-resources/discussions/app-config-form/apps/legacy/LegacyConfigForm.jsx

```diff
@@ -75,7 +75,7 @@ function LegacyConfigForm({
             <DivisionByGroupFields
               onBlur={handleBlur}
               onChange={handleChange}
-              values={values}
+              appConfig={values}
```

Comment:
Do we need to pass the complete appConfig here?

Reply:
We only need 4 values of appConfig, not complete appConfig. It is little bit clean to pass whole object instead of 4+2 = 6 props.